src/lib/templateHelpers.test.ts

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from "chai";
+import {
+  addNamespace,
+  formatForTsDoc,
+  isCommonPathParameter,
+  isCommonQueryParameter,
+} from "./templateHelpers";
+import { commonParameterPositions } from "@commerce-apps/core";
+
+describe("When adding namespaces to individual content (types)", () => {
+  it("Prefixes the namespace successfully ", () => {
+    expect(addNamespace("foo", "types"))
+      .to.be.a("string")
+      .and.equal("types.foo");
+  });
+
+  it("Will not prefix void with a namespace", () => {
+    expect(addNamespace("void", "types")).to.be.a("string").and.equal("void");
+  });
+
+  it("Will not prefix object with a namespace", () => {
```

Comment:
I see duplicate tests with different description, Are you trying to test for case insensitive string "object"?